### ISSUE TITLE
Intermittent failure of psych_sheet_event.html.erb_spec.rb

### ISSUE THREAD

**Issue by jonatanklosko** [cubing/worldcubeassociation.org#754]
Here's rspec output:
```ruby
  1) registrations/psych_sheet_event works

     Failure/Error: expect(rendered).to have_css(".wca-results tbody tr:nth-child(2) .single", text: /\A\s*33.34\s*\z/)

       expected to find css ".wca-results tbody tr:nth-child(2) .single" with text /\A\s*33.34\s*\z/ but there were no matches. Also found "", which matched the selector but not all filters.

     # ./spec/views/registrations/psych_sheet_event.html.erb_spec.rb:61:in `block (2 levels) in <top (required)>'
```
Emerged after #734.

**Comment by jonatanklosko** [comment]
Firstly I rewrote the spec to be cleaner and better and hoped that there could be some tricky hidden bug. But then it failed, moreover both old and new version of the spec failed with the same rspec seed. After quick analysis I got it. Furthermore I mentioned that other specs failed after #734 and it makes sense.
This is more significant than we could thought, thanks to this failing test! So the thing is that our Countries database and virtual Country model **differs**. See:
```ruby
2.3.0 :015 > db = ActiveRecord::Base.connection.execute("SELECT name FROM Countries").to_a.flatten.sort
...
2.3.0 :016 > virtual = Country.all.map(&:name).sort
...
2.3.0 :017 > virtual - db
 => ["Monaco", "Multiple Countries (Asia)", "Multiple Countries (South America)", "Uzbekistan"] 
2.3.0 :018 > db - virtual
 => ["Montserrat"] 
```
I can image that this could have real impact on other pieces of the page.
I'm going to provide a fix. I'm wondering what should be the source of all countries? @jfly I guess you're the one who made virtual ones, where have you taken them from?

**Comment by jfly** [comment]
Damn. They were a database dump at some point in time. Let's just get rid of them?

**Comment by jonatanklosko** [comment]
The inconsistency will be fixed with #741. At the moment I'd suggest to just update Countries seeds (and reseed countries in production?). Changes I propose to apply to the seeds:
- remove "Montserrat" as it belongs to British Overseas Territories and isn't separate country
- add: "Monaco", "Multiple Countries (Asia)", "Multiple Countries (South America)", "Uzbekistan"
Thoughts?
